content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -196,6 +196,111 @@ add a brief description in the :guilabel:`Description` field. Click :guilabel:`F
 the :guilabel:`Rule Configuration` wizard. Now, the new rule is active and devices can connect to
 the Windows virtual :abbr:`IoT (Internet of Things)` box.
 
+Worldline exception
+~~~~~~~~~~~~~~~~~~~
+
+*Worldline* is a payment terminal that can be connected to Odoo's *PoS* (point of sale) system. It
+allows for a comprehensive and fluid payment experience for customers. Worldline is available in
+Benelux (coalition of Belgium, the Netherlands, and Luxembourg).
+
+When using the Windows IoT server to connect the Worldline payment terminal, it is necessary to create
+an exception in the Windows firewall, so that a connection can be made between the Odoo database /
+:abbr:`IoT (Internet of Things)` box and Worldline.
```

Comment:
Line over 100 chars
```suggestion
When using the Windows IoT server to connect the Worldline payment terminal, it is necessary to
create an exception in the Windows firewall so that a connection can be made between the Odoo
database/:abbr:`IoT (Internet of Things)` box and Worldline.
```